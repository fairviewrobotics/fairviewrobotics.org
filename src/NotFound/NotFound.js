import React, { PureComponent } from 'react';

import './NotFound.css';

export default class NotFound extends PureComponent {
  render() {
    return (
      <div>
        <h1 className="title">Page Not Found</h1>
      </div>
    );
  }
}
