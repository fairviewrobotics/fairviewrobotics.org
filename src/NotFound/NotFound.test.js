import NotFound from "./NotFound";
import { shallow } from "enzyme";
import React from "react";

describe('NotFound', () => {
  it('renders without crashing', () => {
    shallow(
      <NotFound/>
    );
  });

  it('should render a Not Found page', () => {
    const wrapper = shallow(
      <NotFound/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});