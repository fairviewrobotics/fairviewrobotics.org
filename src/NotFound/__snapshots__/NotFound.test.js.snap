// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render a Not Found page 1`] = `
<div>
  <h1
    className="title"
  >
    Page Not Found
  </h1>
</div>
`;
